cmd/boost/provider_cmd.go

```diff
@@ -97,3 +109,192 @@ var libp2pInfoCmd = &cli.Command{
 		return nil
 	},
 }
+
+var queryAskCmd = &cli.Command{
+	Name:      "query-ask",
+	Usage:     "Find a miners ask",
+	ArgsUsage: "[minerAddress]",
```

Comment:
In the libp2p-info command we call the argument "provider address". Happy to use either but suggest we change them all to be consistent.

Reply:
I was going to raise that up! I copied the commands from Lotus, but yeah, I also think we should be consistent.. Changing this now.